### PR TITLE
Add electric status

### PR DESCRIPTION
**Pull request by dkmcgowan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 This is a Home Assistant integration for Toyota North America.
 
 ## Stable
-![GitHub release (latest by date)](https://img.shields.io/github/v/release/widewing/ha-toyota-na?style=for-the-badge) ![GitHub Release Date](https://img.shields.io/github/release-date/widewing/ha-toyota-na?style=for-the-badge) ![GitHub Releases](https://img.shields.io/github/downloads/widewing/ha-toyota-na/latest/total?color=purple&label=%20release%20Downloads&style=for-the-badge) 
+![GitHub release (latest by date)](https://img.shields.io/github/v/release/dkmcgowan/ha-toyota-na?style=for-the-badge) ![GitHub Release Date](https://img.shields.io/github/release-date/dkmcgowan/ha-toyota-na?style=for-the-badge) ![GitHub Releases](https://img.shields.io/github/downloads/dkmcgowan/ha-toyota-na/latest/total?color=purple&label=%20release%20Downloads&style=for-the-badge)
```
Comment (widewing) [inline README.md:7]: The badges are referencing your repo, can you change it back to keep consistency?

**Comment by widewing** [comment]
Hi David I don't have EV so cannot verify any EV feature. Also I don't have enough time to maintain this repo recently, can I add you as maintainer?

**Comment by dkmcgowan** [comment]
Sure and I can update the badges or should I change the repo in HACS to
mine?  I've never updated something in HACS, I guess that would mess
updates for people though.

I'm ok with whatever.

David

**Comment by widewing** [comment]
> I guess that would mess updates for people though.

That's also what I worry about, otherwise I'm glad to change the hacs repo and let you take the driver seat.
Anyway I sent you the collaborator invitation

**Comment by dkmcgowan** [comment]
I updated the README, merged the PR, created a new release, and tested the
update from HACS and it all worked so hopefully this will be good for some
that want electric status
David

**Pull request closed without merging**